src/Form/FieldCheckbox.js

```diff
@@ -1,4 +1,4 @@
-import classNames from 'classnames';
+import classNames from 'classnames/dedupe';
```

Comment:
This inherits from RadioButton and have the classes defined here: https://github.com/mesosphere/reactjs-components/pull/279/files#diff-59f6aa2f62c69bf931c3325ad9a5d1d3L166